recipes/Time_Series/Time_Series_Getting_Started.ipynb

```diff
@@ -102,14 +89,16 @@
    "source": [
     "timestamp_column = \"time\"\n",
     "target_columns = [\"total load actual\"]\n",
-    "context_length = 512 # This is set by the pretrained model we will use."
+    "context_length = 512"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Read in the data"
+    "### Read in the data\n",
+    "\n",
+    "We parse the csv into a pandas dataframe, filling in any null values, and clip the dataframe to the context length. Ensure the timestamp column is a datetime."
```

Comment:
can we remove:
```
input_df[timestamp_column] = pd.to_datetime(input_df[timestamp_column])
```

Reply:
> can we remove:
> 
> ```
> input_df[timestamp_column] = pd.to_datetime(input_df[timestamp_column])
> ```

I put this in there to remove the FutureWarning when plotting the target column later (which I can't clear without clearing the actual plot output as well):
```
[/Users/fayvor/Dev/granite-timeseries-cookbook/.venv/lib/python3.12/site-packages/pandas/core/indexes/base.py:7588](https://file+.vscode-resource.vscode-cdn.net/Users/fayvor/Dev/granite-timeseries-cookbook/.venv/lib/python3.12/site-packages/pandas/core/indexes/base.py:7588): FutureWarning: Dtype inference on a pandas object (Series, Index, ExtensionArray) is deprecated. The Index constructor will keep the original dtype in the future. Call `infer_objects` on the result to get the old behavior.
  return Index(sequences[0], name=names)```

I thought it would be taken care of by parse_dates in `pd.read_csv()`, but apparently that is not setting the `dtype` on the column? The column shows up with dtype `object`.

Reply:
I will look into it further, but we don't the unneeded `to_datetime()` call.

Reply:
An alternative is to keep the `to_datetime()` call and remove the `parse_dates` option in `read_csv()`.

Reply:
The error arrises during a pandas `.plot()` call -- they should probably cast as needed internally, as part of that call.

Reply:
The problem was that the dtype of the column was still `object` after reading the csv; `read_csv()` was apparently unable to parse all of the dates.

I've added a `date_format` to the `read_csv()` call to tell it that the incoming dates are timezone-aware. This way it will convert dates to UTC and handle daylight savings time blips.

Reply:
Ok, thanks! It is somewhat surprising this is not handled better in pandas.